api/services/tasks/v1/tasks.proto

```diff
@@ -77,41 +73,32 @@ message CreateTaskRequest {
 }
 
 message CreateTaskResponse {
-	// TODO(stevvooe): We no longer have an id for a task since they are bound
-	// to a single container. Although, we should represent each new task with
-	// an ID so one can differentiate between each instance of a container
-	// running.
-	//
-	// Hence, we are leaving this here and reserving the field number in case
-	// we need to move in this direction.
-	// string id = 1;
-
-	string container_id = 2;
-	uint32 pid = 3;
+	string id = 1;
+	uint32 pid = 2;
 }
 
 message StartTaskRequest {
-	string container_id = 1;
+	string id = 1;
 }
 
 message DeleteTaskRequest {
-	string container_id = 1;
+	string id = 1;
 }
 
 message DeleteResponse {
-	string container_id = 1;
+	string id = 1;
 	uint32 pid = 2;
```

Comment:
Returning these on delete keeps same semantics as waitpid and reduces api calls for clients and helps the restore use case of missed events.